dask/dataframe/core.py

```diff
@@ -2184,13 +2184,14 @@ def meth(self, other, level=None, fill_value=None, axis=0):
 
     @classmethod
     def _bind_comparison_method(cls, name, comparison):
-        """ bind comparison method like DataFrame.add to this class """
+        """ bind comparison method like Series.eq to this class """
 
-        def meth(self, other, level=None, axis=0):
+        def meth(self, other, level=None, fill_value=None, axis=0):
             if level is not None:
                 raise NotImplementedError('level must be None')
             axis = self._validate_axis(axis)
-            return elemwise(comparison, self, other, axis=axis)
+            op = partial(comparison, fill_value=fill_value)
+            return elemwise(op, self, other, axis=axis)
```

Comment:
I'm surprised to learn that we don't pass keyword arguments through `elemwise`.  I suspect that we can resolve this more cleanly later (I'm doing a bit of refactoring on map_partitions and elemwise in https://github.com/dask/dask/pull/4229).  In the meantime I wouldn't mind avoiding the use of `functools.partial` if `fill_value` is None.

Reply:
> I'm surprised to learn that we don't pass keyword arguments through elemwise

I was surprised by that too. It might be worth removing `axis=axis` here, given that it's currently ignored anyways. My thinking is that, without looking closely at `elemwise`, it could be misleading to see keywords being passed to `elemwise` that are ignored. I initially added `fill_value` as a keyword to `elemwise` before realizing that it was being ignored. 

>  In the meantime I wouldn't mind avoiding the use of functools.partial if fill_value is None.

Good idea. I'll add that check.